src/test/java/com/wikia/webdriver/common/core/geoedge/GeoEdgeBrowserMobProxy.java

```diff
@@ -0,0 +1,16 @@
+package com.wikia.webdriver.common.core.geoedge;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * @author Bogna 'bognix' Knychala
+ */
+@Retention(value = RetentionPolicy.RUNTIME)
+@Target(value = {ElementType.METHOD, ElementType.TYPE, ElementType.CONSTRUCTOR})
+public @interface GeoEdgeBrowserMobProxy {
```

Comment:
I want to get rid of this old class eventually, so the name doesn't matter. I'm just updating Oasis for now, Mercury and WikiaMobile are planned next.